README.md

```diff
@@ -1,33 +1,28 @@
-# TERRAFORM AWS INTEGRATIONS
+# Honeycomb Terraform AWS Integrations
 
 [![OSS Lifecycle](https://img.shields.io/osslifecycle/honeycombio/terraform-aws-integrations)](https://github.com/honeycombio/home/blob/main/honeycomb-oss-lifecycle-and-practices.md)
 
-
-This repo contains a set of modules in
-the [modules folder](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules) for resources
-in [AWS](https://aws.amazon.com/) using [Terraform](https://www.terraform.io/) to send observability data
-to [Honeycomb](https://www.honeycomb.io/).
+This repo contains a set of modules in the [modules folder](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules) for resources in [AWS](https://aws.amazon.com/) using [Terraform](https://www.terraform.io/) to send observability data to [Honeycomb](https://www.honeycomb.io/).
 
 ## How does this work?
 
 ![AWS Integrations architecture](https://github.com/honeycombio/terraform-aws-integrations/blob/main/docs/overview.png?raw=true)
 
-**Integrations supported are:**
-
-* [CloudWatch Logs](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/cloudwatch-logs)
+## Supported Integrations
 
+Available supported integrations include:
 
+* [CloudWatch Logs](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/cloudwatch-logs)
 * [CloudWatch Metrics](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/cloudwatch-metrics)
-
-
 * [RDS Logs](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/rds-logs)
-
-
 * [Logs from a S3 Bucket](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/s3-logfile)
 
-## Use
+To install individual integrations, refer to each integration's README.
+Otherwise, refer to the [minimal configuration](#usage) to turn on all supported integrations.
+
+## Usage
 
-The minimal config to turn all integrations on is:
+The minimal Terraform configuration to turn on all [supported integrations](#supported-integrations) is:
```

Comment:
Is there a specific file or interface where this configuration should go?

Reply:
given the repo is about terraform modules - do we need to say terraform config? re: file - not really folks can include in their IAC *.tf or add this into the right spot that they will determine